c54094821.lua

```diff
@@ -21,7 +21,8 @@ function c54094821.activate(e,tp,eg,ep,ev,re,r,rp)
 	e1:SetCode(EFFECT_CANNOT_ACTIVATE)
 	e1:SetTargetRange(0,1)
 	e1:SetValue(c54094821.aclimit)
-	if Duel.GetTurnPlayer()==tp then
+	if Duel.GetTurnPlayer()==tp and Duel.GetCurrentPhase()>PHASE_STANDBY
```

Comment:
If it is activated during your own Draw Phase, it will still be reseted in your next Turn.